bears/natural_language/WriteGoodLintBear.py

```diff
@@ -11,7 +11,7 @@
         )
 class WriteGoodLintBear:
     """
-    Lints the text files using ``write-good`` for improving proses.
+    Lints the text files using "write-good" for improving proses.
```

Comment:
`write-good` is a technical term - it is the name of the program used.  It shouldnt be replaced with textual quotes